From 425b58355eaab88492588e29510f2785b987f2dc Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Thu, 25 Jan 2024 17:11:25 +0100
Subject: [PATCH 0693/1695] dts: stm32: update name of RCC pll node on
 STM32MP25 boards

Change the PLL node name, replace st,pll@${X} by st,pll-${X+1}.

Fix the following DT dtc warnings for STM32MP25 boards:

Warning (unit_address_vs_reg): <pll node path> \
  node has a unit name, but no reg property

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git b0a490224464112600253b1cdb5ea4ae1551b265 dts: stm32: update name of RCC pll node on STM32MP25 boards]
Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Change-Id: I40786bae4d62043fc06f722e14f6d69bdf50c045
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/354516
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 .../arm/dts/stm32mp257f-dk-ca35tdcid-rcc.dtsi    | 16 ++++++++--------
 .../arm/dts/stm32mp257f-ev1-ca35tdcid-rcc.dtsi   | 16 ++++++++--------
 2 files changed, 16 insertions(+), 16 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp257f-dk-ca35tdcid-rcc.dtsi b/core/arch/arm/dts/stm32mp257f-dk-ca35tdcid-rcc.dtsi
index 3d49d8a39..9529f3c2d 100644
--- a/core/arch/arm/dts/stm32mp257f-dk-ca35tdcid-rcc.dtsi
+++ b/core/arch/arm/dts/stm32mp257f-dk-ca35tdcid-rcc.dtsi
@@ -114,7 +114,7 @@
 		MCO_CFG(MCO2, MUX_MCO2_FLEX62, MCO_OFF)
 	>;
 
-	pll1: st,pll@0 {
+	pll1: st,pll-1 {
 		st,pll = <&pll1_cfg_1200Mhz>;
 
 		pll1_cfg_1200Mhz: pll1-cfg-1200Mhz {
@@ -128,7 +128,7 @@
 		};
 	};
 
-	pll2: st,pll@1 {
+	pll2: st,pll-2 {
 		st,pll = <&pll2_cfg_600Mhz>;
 
 		pll2_cfg_600Mhz: pll2-cfg-600Mhz {
@@ -137,7 +137,7 @@
 		};
 	};
 
-	pll3: st,pll@2 {
+	pll3: st,pll-3 {
 		st,pll = <&pll3_cfg_800Mhz>;
 
 		pll3_cfg_800Mhz: pll3-cfg-800Mhz {
@@ -151,7 +151,7 @@
 		};
 	};
 
-	pll4: st,pll@3 {
+	pll4: st,pll-4 {
 		st,pll = <&pll4_cfg_1200Mhz>;
 
 		pll4_cfg_1200Mhz: pll4-cfg-1200Mhz {
@@ -160,7 +160,7 @@
 		};
 	};
 
-	pll5: st,pll@4 {
+	pll5: st,pll-5 {
 		st,pll = <&pll5_cfg_532Mhz>;
 
 		pll5_cfg_532Mhz: pll5-cfg-532Mhz {
@@ -169,7 +169,7 @@
 		};
 	};
 
-	pll6: st,pll@5 {
+	pll6: st,pll-6 {
 		st,pll = <&pll6_cfg_500Mhz>;
 
 		pll6_cfg_500Mhz: pll6-cfg-500Mhz {
@@ -178,7 +178,7 @@
 		};
 	};
 
-	pll7: st,pll@6 {
+	pll7: st,pll-7 {
 		st,pll = <&pll7_cfg_835_51172Mhz>;
 
 		pll7_cfg_835_51172Mhz: pll7-cfg-835-51172Mhz {
@@ -188,7 +188,7 @@
 		};
 	};
 
-	pll8: st,pll@7 {
+	pll8: st,pll-8 {
 		st,pll = <&pll8_cfg_594Mhz>;
 
 		pll8_cfg_594Mhz: pll8-cfg-594Mhz {
diff --git a/core/arch/arm/dts/stm32mp257f-ev1-ca35tdcid-rcc.dtsi b/core/arch/arm/dts/stm32mp257f-ev1-ca35tdcid-rcc.dtsi
index 6bb42fea5..b0199e5e5 100644
--- a/core/arch/arm/dts/stm32mp257f-ev1-ca35tdcid-rcc.dtsi
+++ b/core/arch/arm/dts/stm32mp257f-ev1-ca35tdcid-rcc.dtsi
@@ -114,7 +114,7 @@
 		MCO_CFG(MCO2, MUX_MCO2_FLEX62, MCO_OFF)
 	>;
 
-	pll1: st,pll@0 {
+	pll1: st,pll-1 {
 		st,pll = <&pll1_cfg_1200Mhz>;
 
 		pll1_cfg_1200Mhz: pll1-cfg-1200Mhz {
@@ -128,7 +128,7 @@
 		};
 	};
 
-	pll2: st,pll@1 {
+	pll2: st,pll-2 {
 		st,pll = <&pll2_cfg_600Mhz>;
 
 		pll2_cfg_600Mhz: pll2-cfg-600Mhz {
@@ -137,7 +137,7 @@
 		};
 	};
 
-	pll3: st,pll@2 {
+	pll3: st,pll-3 {
 		st,pll = <&pll3_cfg_800Mhz>;
 
 		pll3_cfg_800Mhz: pll3-cfg-800Mhz {
@@ -151,7 +151,7 @@
 		};
 	};
 
-	pll4: st,pll@3 {
+	pll4: st,pll-4 {
 		st,pll = <&pll4_cfg_1200Mhz>;
 
 		pll4_cfg_1200Mhz: pll4-cfg-1200Mhz {
@@ -160,7 +160,7 @@
 		};
 	};
 
-	pll5: st,pll@4 {
+	pll5: st,pll-5 {
 		st,pll = <&pll5_cfg_532Mhz>;
 
 		pll5_cfg_532Mhz: pll5-cfg-532Mhz {
@@ -169,7 +169,7 @@
 		};
 	};
 
-	pll6: st,pll@5 {
+	pll6: st,pll-6 {
 		st,pll = <&pll6_cfg_500Mhz>;
 
 		pll6_cfg_500Mhz: pll6-cfg-500Mhz {
@@ -178,7 +178,7 @@
 		};
 	};
 
-	pll7: st,pll@6 {
+	pll7: st,pll-7 {
 		st,pll = <&pll7_cfg_835_51172Mhz>;
 
 		pll7_cfg_835_51172Mhz: pll7-cfg-835-51172Mhz {
@@ -188,7 +188,7 @@
 		};
 	};
 
-	pll8: st,pll@7 {
+	pll8: st,pll-8 {
 		st,pll = <&pll8_cfg_594Mhz>;
 
 		pll8_cfg_594Mhz: pll8-cfg-594Mhz {
-- 
2.47.2

